File: Python/0024. The Simple Problem.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    
    while True:
        try:
            a,k,n = map(int,input().strip().split())
            print((n+1)*a+n*(n+1)//2*k)
        except EOFError:
            break

    
if __name__ == "__main__":
    func()

File: C++/0024. The Simple Problem.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl;     

    int a,k,n;
    while(cin>>a>>k>>n){
        cout<<((n+1)*a+n*(n+1)/2*k)<<endl;
    }
    return 0;
}
